ethereum.md

```diff
@@ -122,31 +253,61 @@ To do so, we'll extend sort `JSON` with some EVM specific syntax, and provide a
     syntax Bool ::= "#hasPost?" "(" JSON ")" [function]
  // ---------------------------------------------------
     rule #hasPost? ({ .JSONList }) => false
-    rule #hasPost? ({ (KEY:String) : _ , REST }) => KEY ==String "post" orBool #hasPost? ({ REST })
+    rule #hasPost? ({ (KEY:String) : _ , REST }) => (KEY in #postKeys) orBool #hasPost? ({ REST })
```

Comment:
you seem to be using postKeys in another place to handle assertions, which is causing the tests to fail because it doesn't know how to handle "expect", "expet", and "export" anymore. Note that the reason I made this change is that those three keys actually do /not/ signify that the test is passing, which was making tests fail on the commit hash I inherited from you.